content/docs/user-guide/scripting/lua/debugging-tutorial.md

```diff
@@ -1,17 +1,17 @@
 ---
-linkTitle: Debugging with the Lua Editor
-title: Debugging with the Lua Editor
-description: Use the Lua Editor to debug Lua scripts in Open 3D Engine.
+linkTitle: Debugging with Lua Editor
+title: Debugging with Lua Editor
+description: Use Lua Editor to debug Lua scripts in Open 3D Engine (O3DE).
 toc: true
 weight: 900
 ---
 
-O3DE **Lua Editor** (Lua IDE) offers an intuitive integrated development environment (IDE) that makes it easy to author, debug, and edit Lua scripts when you create or extend your game. Lua Editor is a standalone application, but can be opened directly from **O3DE Editor** using the Tools menu.
-
-## Tutorial: Using Lua Editor for debugging with O3DE Editor 
+O3DE **Lua Editor** (Lua IDE) offers an intuitive integrated development environment (IDE) that makes it easy to author, debug, and edit Lua scripts when you create or extend your game. Lua Editor is a standalone application, but you can open it directly in **O3DE Editor** from the Tools menu.
 
 This tutorial shows you how to use O3DE Lua Editor to perform debugging operations on a sample script.
 
+## Create an entity with a Lua Script component
```

Comment:
The section title puts too much emphasis on creating an entity, in my opinion. How about:
```suggestion
## Add sample Lua Script to an entity
```